internal/querycoordv2/balance/utils.go

```diff
@@ -0,0 +1,51 @@
+package balance
+
+import (
+	"context"
+	"time"
+
+	"github.com/milvus-io/milvus/internal/querycoordv2/task"
+)
+
+func CreateSegmentTasksFromPlans(ctx context.Context, checkerID int64, timeout time.Duration, plans []SegmentAssignPlan) []task.Task {
+	ret := make([]task.Task, 0)
+	for _, p := range plans {
+		actions := make([]task.Action, 0)
```

Comment:
add a constraint to check to and from should be -1 at the same time

Reply:
> add a constraint to check to and from should be -1 at the same time

This kind plan will never not be created.

Reply:
there is no code to guarantee, it is better to add a constraint to check